indico/modules/categories/templates/management/roles.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'categories/management/base.html' %}
+{% from 'events/roles/_roles.html' import render_roles %}
+
+{% block title %}{% trans %}Roles Setup{% endtrans %}{% endblock %}
+
+{% block content %}
+    <table class="i-table-widget multi-tbody" id="event-roles">
+        {{ render_roles(roles, collapsed=true, email_button=False) }}
```

Comment:
`false` (jinja uses lowercase true/false/none, even though the python-style versions work as well)

Reply:
and `collapsed=true` isn't needed here